src/formattedText.py

```diff
@@ -81,7 +81,8 @@ def printText(self, y, x, printer, maxLen):
             if index % 2 == 1:
                 # text
                 toPrint = val[0:maxLen - printedSoFar]
-                printer.screen.addstr(y, x + printedSoFar, toPrint)
+                printer.addstr(y, x + printedSoFar, toPrint,
+                               ColorPrinter.CURRENT_COLORS)
```

Comment:
ahh nice. what do you think about a sugar method like `printer.addCurrentColorStr`?